applications/FluidDynamicsApplication/tests/adjoint_vms_element_2d.py

```diff
@@ -146,7 +156,7 @@ def test_CalculateFirstDerivativesLHS1(self):
         self._assign_solution_step_data1(0)
         self._assign_solution_step_data2(1)
 
-    def test_CalculateFirstDerivativesLHS2(self):
+    def testCalculateFirstDerivativesLHS2(self):
```

Comment:
I personally would prefer the other name since its more readable

Just my opinion though